internal/server/cache/cache.go

```diff
@@ -117,13 +123,17 @@ func NewCache(
 		c.svgSearchIndex = hierarchy.BuildStatVarSearchIndex(c.rawSvgs, c.parentSvgs, c.blocklistSvgs)
 	}
 
-	if options.CustomProv {
-		customProv, err := sqlquery.GetProvenances(store.SQLClient)
+	if options.CacheSQL {
+		sqlProv, err := sqlquery.GetProvenances(store.SQLClient)
+		if err != nil {
+			return nil, err
+		}
+		c.sqlProvenances = sqlProv
+		sqlExistenceMap, err := sqlquery.EntityVariable(store.SQLClient)
 		if err != nil {
 			return nil, err
 		}
-		c.customProvenances = customProv
+		c.sqlEntityVariable = sqlExistenceMap
```

Comment:
Actually `sqlExistenceMap` might be a good name for the cache entry itself.

Reply:
Done